frontend/packages/dev-console/src/components/health-checks/HealthChecks.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { PageHeading } from '@console/internal/components/utils';
+import FormSection from '../import/section/FormSection';
+import HealthCheckProbe from './HealthCheckProbe';
+import { HealthChecksProbeType } from './health-checks-types';
+import { getHealthChecksProbeData } from './health-checks-utils';
+
+interface HealthChecksProps {
+  pageTitle?: string;
+}
+
+const renderProbes = () => {
+  return (
+    <>
+      <HealthCheckProbe
```

Comment:
Why do you have  three `Formik` instances for this form here? How would you integrate the forms inside the actual form or advanced options of git import if you have created separate formik instances of each section?